document/CoreFormat.mdk

```diff
@@ -668,10 +672,10 @@ the MIME content-type (RFC 2045) of the file.
 ### `region` property { #physicalLocationComponent-region }
 
 A `physicalLocationComponent` object may have a property named `region` whose value is a `region` object (&sect;[#region]),
-that represents the region within a file where the issue was detected.
+that represents the region within a file where the result occurs.
```

Comment:
@michaelcfanning I looked at all occurrences (hah) of "occur" and its various forms. In this particular case, and in all other cases where we're talking about a property of a particular result, I've changed it to "occurred" (_e.g._, "where the result occurred"). In other context, "occurs" seems better to me (_e.g._, "... the location ... where the class name occurs").